SimplyPluralApi.yml

```diff
@@ -1397,106 +1172,86 @@ paths:
         "404":
           description: NOT FOUND
 
-  /v1/friend/{userId}:
+  /v1/friend/{friendId}:
     patch:
       security:
         - APITokenAuth: []
       summary: "Update friend settings"
       description: "Update your settings for a friend by Id"
+      parameters:
+        $ref: "./SimplyPluralApiCommons.yml#/components/parameters/friendId"
       tags:
         - Friends
-      parameters:
-        - in: path
-          name: userId
-          schema:
-            type: string
-          required: true
-          description: String Id of the friend user account
       responses:
         "200":
           description: OK
         "404":
           description: NOT FOUND
-  /v1/friend/{userId}/{friendId}:
+  /v1/friend/{systemId}/{friendId}:
     get:
       security:
         - APITokenAuth: []
       summary: "Get friend settings"
-      description: "Get a friend from a user (Typically your user)"
+      description: "Get a friend from a user"
       tags:
         - Friends
       parameters:
         - in: path
-          name: userId
+          name: systemId
           schema:
             type: string
           required: true
-          description: String Id of the origin friend (Typically you)
+          description: The Id of the origin friend (Typically you). This can be a mongo Id but also a plain string (backwards-compatibility).
         - in: path
           name: friendId
           schema:
             type: string
           required: true
-          description: String Id of the destination friend (Typically your friend)
+          description: The Id of the destination friend (Typically your friend). This can be a mongo Id but also a plain string (backwards-compatibility).
```

Comment:
friendIds/userIds are never mongodb Ids. They're just plain strings of random characters.